projects/components/src/table/controls/table-controls-api.ts

```diff
@@ -35,6 +35,10 @@ export type TableControlOption<T = unknown> =
   | TableFilterControlOption<T>
   | TablePropertyControlOption<T>;
 
+export type LabeledTableControlOption = TableControlOption & {
```

Comment:
Does it make sense to put this in the api? Removing the label from the api (since it's only used by dashboarding) was kind of the goal of that refactor

Reply:
Ah, I see what you were doing there now. I like it. I'll move it back. One sec.